contrib/platform/test/com/sun/jna/platform/win32/Kernel32Test.java

```diff
@@ -539,21 +539,6 @@ public void testCreateProcess() {
         assertTrue(processInformation.dwProcessId.longValue() > 0);
     }
 
-    public void testGetEnvironmentVariable() {
```

Comment:
On "don't hold it against me" I can [recommend an excellent book](http://art.dblock.org/2015/01/21/jennifer-doyle-hold-it-against-me.html).

![](http://art.dblock.org/images/posts/2015-01-21-jennifer-doyle-hold-it-against-me/jennifer-doyle-hold-it-against-me.jpg)